MapboxVision.podspec

```diff
@@ -24,8 +24,8 @@ Pod::Spec.new do |s|
 
   s.swift_version = '4.1'
 
-  s.dependency "MapboxVisionCore", "= 0.2.0"
+  s.dependency "MapboxVisionCore", "~> 0.3.0"
```

Comment:
No need to update this dependency.

Reply:
I made it this a `minor` bump because the dependent SDK, `"MapboxMobileEvents"` have gone through a series of minor releases `~> 0.6.0` - `~> 0.8.1` since its last update. This needs to be a `0.3.0`.

Reply:
`MapboxVisionCore` is unaffected since it has no external dependencies except for `opencv`.
What you're probably referencing is a version of `MapboxVision`. I don't see a reason for making a minor update to `MapboxVision` since its api hasn't changed.

Reply:
@chezzdev actually that's not correct, `MapboxVision` depends on `MapboxMobileEvents` as shown here on [line#29](https://github.com/mapbox/mapbox-vision-ios/pull/54/files#diff-5a0af8c40b8e7c3bb45e642e104e149dR29) in the MapboxVision.podspec file

Reply:
Yeah, I agree. VisionCore doesn't need a version bump. Reverted in recent commit.